configs/components/augeas.rb

```diff
@@ -37,7 +39,11 @@
       pkg.build_requires 'runtime'
     end
 
-    pkg.build_requires 'pkgconfig'
+    if platform.architecture == "s390x"
+      pkg.environment "PATH" => "/opt/pl-build-tools/bin:$$PATH:#{settings[:bindir]}"
+      pkg.environment "CFLAGS" => settings[:cflags]
+      pkg.environment "LDFLAGS" => settings[:ldflags]
```

Comment:
Probably out of scope for this PR, but why don't we set `:CFLAGS` and `:LDFLAGS` for everything everywhere all the time forever?